### DISCUSSION TITLE
is it possible to mix cartoon styles and image prompt?

### DISCUSSION THREAD

**Discussion by puntomaupunto** [lllyasviel/Fooocus#2838] [Q&A] [answered]
Hi! 
I am playing with styles, but I got stumped. If I prompt "a young happy woman, glasses" with SAI Line Art style (or other styles, removing all default ones) I got a very nice result. But it seems that it does not work if I add a image prompt: the result is definitely not line art, unless I use Pyracanny (but in that case I cannot add specifications to the image) . Is it a limitation of fooocus (in that case, never mind) or I am forgetting something?

**Comment by xhoxye** [comment]
Image prompt include styles

**Comment by ILikeToasters** [comment, marked as answer]
It's not really a limitation, ImagePrompt is designed to bring over the look of the image. Pyrocanny and cpds bring over the structure.
